cmd/posix.go

```diff
@@ -55,16 +56,17 @@ func checkPathLength(pathName string) error {
 	// Convert any '\' to '/'.
```

Comment:
checkPathLength can be renamed to isPathValid because it's doing more than path length check